### ISSUE TITLE
error (using renpy 8.1)

### ISSUE THREAD

**Issue by codachrome-png** [CoolerMudkip/Ren-py-Animalese#2]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/speak.rpy", line 8, in script
    init python:
  File "game/speak.rpy", line 15, in <module>
    from pydub.playback import play
ImportError: cannot import name 'play' from 'pydub.playback' (pydub/playback.py)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/speak.rpy", line 8, in script
    init python:
  File "D:\renpy\renpy-7.4.11-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy\renpy-7.4.11-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/speak.rpy", line 15, in <module>
    from pydub.playback import play
ImportError: cannot import name 'play' from 'pydub.playback' (pydub/playback.py)

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.1.23060707
IsekaiWIP 1.0
Wed Aug  9 18:58:02 2023

**Comment by ztc0611** [comment]
Yeah, it doesn't work on modern Ren'py anymore. I should get around to updating this eventually, but if someone wants to cut me in line feel free to fork and do as you please.
